File: webapi/Classes/Location.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace webapi.Classes
{
    public class Location
    {
        [Key]
        [Required]
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string PostalCode { get; set; }
        public string Country { get; set; }
        public string Website { get; set; }
        public string PhoneNumber { get; set; }
        public string EmailAddress { get; set; }

    }
}

File: angularapp/src/interfaces/location.ts
export interface Location {
  id: string;
  name: string;
  address: string;
  city: string;
  postalCode: string;
  country: string;
  website: string;
  phoneNumber: string;
  emailAddress: string;
  zones: Zone[];
}
export interface Zone {
  zoneName: string;
  numberOfSeats: number;
}
